functions/poll.js

```diff
@@ -276,6 +277,42 @@ async function onIntakeCompleted(id, fields, meta) {
   return {};
 }
 
+async function checkVolunteers(table) {
```

Comment:
was originally going to use `pollTable` but as @maybejustmaybe noted this was discarded due to it not account for null status.